src/state/slices/opportunitiesSlice/resolvers/cosmosSdk/index.ts

```diff
@@ -60,3 +70,91 @@ export const cosmosSdkOpportunityIdsResolver = async ({
     data: uniqueValidatorAccountIds,
   }
 }
+
+export const cosmosSdkStakingOpportunitiesMetadataResolver = async ({
+  opportunityIds: validatorIds,
+  opportunityType,
+  reduxApi,
+}: OpportunitiesMetadataResolverInput): Promise<{
+  data: GetOpportunityMetadataOutput
+}> => {
+  const stakingOpportunitiesById: Record<StakingId, OpportunityMetadata> = {}
+
+  const state = reduxApi.getState() as ReduxState
+
+  const { CosmosSdkOpportunitiesAbstraction } = selectFeatureFlags(state)
+
+  // TODO(gomes): default validatorIds
```

Comment:
Implement me before opening

- No ValidatorIds length -> use both default validators for Cosmos and Osmosis
- Has length -> check if there is an ATOM/OSMO AccountId

Both can be abstracted as two `hasCosmosValidatorId` and `hasOsmosisValidatorId`, `false` for both for the first case, `boolean` for the latter

Reply:
this business logic around default validators has always been incredibly confusing to me at least, let's chat and confirm requirements with @DiggyDiggy2 @reallybeard and maybe even write a (gasp) test for it with this fresh implementation

Reply:
Sounds good to me, safer with thorough tests and less fluff in this PR as well if we PR it in isolation - I'll add it as a follow-up after we confirm requirements